packages/stable/src/api/documents/types.ts

```diff
@@ -0,0 +1,189 @@
+import { CogniteExternalId, CogniteInternalId } from '@cognite/sdk-core';
+
+export type DocumentSearchRequest = DocumentFilterOption &
+  DocumentSearch &
+  DocumentAggregates &
+  DocumentSort &
+  DocumentSearchLimit;
+
+export interface DocumentFilterOption {
+  filter?: DocumentFilter;
+}
+
+export interface DocumentSearch {
+  search?: { query?: string; highlight?: boolean };
+}
+
+export interface DocumentAggregates {
+  aggregates?: DocumentCountAggregate[];
+}
+
+export interface DocumentSort {
+  sort?: DocumentSortItem[];
+}
+
+export type DocumentSortOrder = 'asc' | 'desc';
+
+export type DocumentSortItem = DocumentFilterProperty & {
+  order: DocumentSortOrder;
+};
+
+export interface DocumentSearchLimit {
+  limit?: number;
+}
+
+export interface Highlight {
```

Comment:
Is it intentional that this does not have the `Document` prefix as it has in the service spec?

Reply:
I've added the Document prefix now